voice/methods/recordings/putCallsCallIdRecording.md

```diff
@@ -33,6 +33,19 @@ curl -X PUT \
      }'
 ```
 
+{% sample lang="java" %}
+
+```java
+ModifyCallRecordingState modifyCallRecordingState = new ModifyCallRecordingState();
+modifyCallRecordingState.setState(State1Enum.PAUSED);
+
+try {
+    ApiResponse<Void> response = voiceClient.modifyCallRecordingState(VOICE_ACCOUNT_ID, "callId", modifyCallRecordingState);
```

Comment:
Why `Void`? Seems a bit odd. If no response is expected I'm fine with just having the method call (that's what I did for the other languages)

Reply:
The `Void` is the `response.Data` but the response still has a `StatusCode` `Headers` etc.  This is just how APIMatic designed the strong type.

TDLC:
A response is expected but it has no `Data` more of an acknowledgment like 200

Reply:
Aaah makes sense